articles/cognitive-services/Bing-Web-Search/quickstarts/nodejs.md

```diff
@@ -22,102 +22,145 @@ Use this quickstart to make your first call to the Bing Web Search API and recei
 Here are a few things that you'll need before running this quickstart:
 
 * [Node.js 6](https://nodejs.org/en/download/) or later
-* A subscription key  
+* A subscription key
 
 ## Create a project and declare required modules
 
-Create a new Node.js project in your favorite IDE or editor. Then copy the code snippet below into your project. This quickstart uses strict mode and requires the `https` module to send and receive data.
+Create a new Node.js project in your favorite IDE or editor.
+Then copy the code snippet below into your project in a file named `search.js`.
+First we require `https` from the standard library, we'll use it to send and receive data.
 
 ```javascript
-// Use strict mode.
-'use strict';
+// Usage:
+//
+//   node search.js "My awesome query"
+//
+const https = require('https')
+```
+
+## Set the subscription key
+
+This code looks up the environment variable, `AZURE_SUBSCRIPTION_KEY`,
```

Comment:
Using the env variable is a valid approach and I'm in agreement that it belongs in these quickstart guides, however, as you mentioned in your PR, this has the potential to create friction for novice developers. What if we made a slight adjustment so that it read like this:

This code snippet uses the `AZURE_SUBSCRIPTION_KEY` environment variable to store your subscription key, a good practice to prevent the accidental exposure of your secret keys when deploying code. If you're unfamiliar with using environment variables, or looking to run this app as fast as possible, then you can replace `process.env['AZURE_SUBSCRIPTION_KEY']` with your subscription key set as a string.

* As soon as we have an article on setting and using environment variables we'll add a link.